src/main/asciidoc/overview/overview008.adoc

```diff
@@ -236,7 +237,7 @@ specification. The Jakarta EE 8 platform requires Managed Beans
 
 Jakarta Contexts and Dependency Injection (CDI) defines a set of
```

Comment:
A doubt about the CDI (and others like JMS) acronym, is it still used in Jakarta?